ui/app/src/components/CreateFolderDialog/CreateFolderContainer.tsx

```diff
@@ -132,7 +165,17 @@ export function CreateFolderContainer(props: CreateFolderContainerProps) {
 										setConfirm({ body: message });
 									} else {
 										if (rename) {
-											onRenameFolder(site, path, name);
+											if (containsItemsInWorkflow) {
+												const packageIds: number[] = packagesInWorkflow.map((pkg) => pkg.id);
+												cancelPackages(site, {
+													packageIds,
+													comment: `Cancel packages to rename folder "${path}"`
```

Comment:
@rart right now the comment is 'generated'. Should it have i18n?

Reply:
Comment and package name generation in general is something we need to discuss in detail and adjust in all the recent publishing work. I've added a task on our tracker ticket. And let's leave a TODO here.
```suggestion
													// TODO: Correct comment generation
													comment: `Cancel packages to rename folder "${path}"`
```